src/ucp/tag/offload.h

```diff
@@ -59,7 +59,8 @@ int ucp_tag_offload_post(ucp_context_t *ctx, ucp_request_t *req);
 static UCS_F_ALWAYS_INLINE void
 ucp_tag_offload_try_post(ucp_context_t *ctx, ucp_request_t *req)
 {
-    if (ucs_unlikely(req->recv.length >= ctx->tm.offload.thresh)) {
+    if (ucs_unlikely((req->recv.length >= ctx->tm.offload.thresh) &&
```

Comment:
can you explain the fix (preferably amend the commit message + push force)?

Reply:
added explanation to the commit message.